tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
```

Comment:
Good call, though I did it with a twist -- I went ahead and modified `CONTRIBUTING.md`, since (presumably) anyone contributing to this package is going to want to know how to write tests.